pages/your_role/researcher.md

```diff
@@ -1,39 +1,53 @@
 ---
 title: Researcher
-keywords: [Keyword 1,Keyword 2, Keyword n]
-summary: Fill in your summary or delete this property when not needed
-contributors: [contributor1, contributor2]
+keywords: [data management, researcher, training, get started]
+contributors: [abotzki]
```

Comment:
```suggestion
contributors: [Alexander Botzki]
```
The full name is used in the metadata of the page. You can link your github in the [CONTRIBUTORS file](https://github.com/elixir-europe/rdm-toolkit/blob/master/_data/CONTRIBUTORS.yaml)